packages/form-js-carbon-styles/src/carbon-styles.js

```diff
@@ -0,0 +1,993 @@
+
+          &,
+          &::placeholder {
+            color: var(--cds-text-primary);
+          }
+        }
+
+        .fjs-input.flatpickr-input:disabled {
+          color: var(--cds-text-disabled);
+        }
+
+        .fjs-input.flatpickr-input:disabled::placeholder {
+          color: var(--cds-text-disabled);
+        }
+
+        .fjs-input.flatpickr-input:disabled::placeholder {
+          color: var(--cds-text-disabled);
+        }
+
+        select {
+          ${getBaseInputStyles({ height: '1.5rem' })};
+          ${getSelectArrowStyles({
+    arrowRightPosition: 'var(--cds-spacing-03)',
+    color: theme.iconPrimary,
+  })};
+          border-bottom: none;
+          padding-right: 2rem;
+        }
+
+        .fjs-input-group {
+          display: flex;
+          flex-direction: row-reverse;
+          position: relative;
+        }
+
+        .fjs-input-group .fjs-input-adornment {
+          border: none;
+          border-radius: 0;
+          background-color: var(--cds-field, var(--cds-field-01));
+          display: flex;
+          padding-right: var(--cds-spacing-05);
+        }
+
+        .fjs-input-group .fjs-input-adornment svg {
+          color: var(--cds-icon-primary);
+          cursor: pointer;
+        }
+
+        .flatpickr-wrapper {
+          height: 100%;
+          position: initial;
+        }
+
+        .fjs-timepicker.fjs-timepicker-anchor {
+          position: unset;
+        }
+
+        .flatpickr-calendar.static {
+          top: calc(100% + 3px);
+        }
+
+        .flatpickr-calendar .flatpickr-prev-month svg,
+        .flatpickr-calendar .flatpickr-next-month svg {
+          height: 16px;
+        }
+
+        .flatpickr-day.today {
+          position: relative;
+          color: var(--cds-link-primary);
+          font-weight: 600;
+          border-color: transparent;
+        }
+
+        .flatpickr-day.selected,
+        .flatpickr-day.today.selected,
+        .flatpickr-day.selected:hover,
+        .flatpickr-day.today.selected:hover {
+          background-color: var(--cds-button-primary);
+          color: var(--cds-text-on-color);
+        }
+
+        .flatpickr-day:focus {
+          outline: 2px solid var(--cds-focus);
+          outline-offset: -2px;
+        }
+
+        .flatpickr-day.selected:focus {
+          outline: 0.0625rem solid var(--cds-focus);
+          outline-offset: -0.1875rem;
+        }
+
+        .flatpickr-day:hover {
+          background: var(--cds-layer-hover, var(--cds-layer-hover-01));
+        }
+
+        .flatpickr-days,
+        .flatpickr-weekdays {
+          padding: unset;
+          width: unset;
+        }
+      }
+    }
+  `}
+`;
+
+const SELECT_STYLES = css`
+  ${({ theme }) => css`
+    .fjs-container {
+      .fjs-form-field-select .fjs-input-group {
+        ${getBaseInputStyles({ height: '2.5rem' })}
+        ${getSelectArrowStyles({
+    arrowRightPosition: 'var(--cds-spacing-05)',
+    color: theme.iconPrimary,
+  })}
```

Comment:
Very weird formatting :sweat_smile:

Reply:
Not sure which problem eslint has with this line 😅